IDEAS/BoundaryConditions/Interfaces/PartialSimInfoManager.mo

```diff
@@ -247,12 +247,12 @@ protected
     annotation (Placement(transformation(extent={{-86,136},{-78,144}})));
 initial equation
   V_tot=volumePort.V_tot;
-  A_tot=areaPort.A_tot;
+  A_tot=max(0.001,areaPort.A_tot); //max(.,.) to avoid division by 0 error when no surfaces with possible air leakage
```

Comment:
```suggestion
  A_tot=max(Modelica.Constants.small,areaPort.A_tot); //max(.,.) to avoid division by 0 error when no surfaces with possible air leakage
```